bot/templates/edit-chatbot.html

```diff
@@ -11,11 +11,11 @@
 </head>
 <body class="bg-gray-100">
     <!-- Navbar -->
-    {% set title = 'Edit Chatbot' %}
     {% include 'navbar.html'  %}
 
     <main class="container mx-auto mt-8 px-4">
         <form id="chatbot-form" method="POST" class="bg-white shadow-md rounded-lg p-6" onsubmit="submitForm(event, '{{ bot.id if bot.id else null }}')">
+            <h1 class="text-2xl font-bold text-center mb-6">Edit Chatbot</h1>

```

Comment:
Why do you need to change this?

Reply:
I  want to use the navbar to switch between the chatbot and users pages, instead of displaying the current form title in the navbar.